v1/chatbot/combine_detection_logic.py

```diff
@@ -80,7 +80,9 @@ def combine_output_of_detection_logic_and_tag(entity_data, text):
     for entity, entity_list in entity_data.iteritems():
         if entity_list:
             for entity_identified in entity_list:
-                if entity_identified[ORIGINAL_TEXT] and entity_identified[DETECTION_METHOD] in [FROM_MESSAGE]:
+                if entity_identified[ORIGINAL_TEXT] and entity_identified[DETECTION_METHOD] in [FROM_MESSAGE,
+                                                                                                MODEL_VERIFIED,
```

Comment:
I think name 'MODEL_VERIFIED' looks off. 'FROM_MESSAGE' is the source from where entity is detected, model is the method.